arbitrum-docs/devs-how-tos/how-to-get-l2block-on-l1.mdx

```diff
@@ -0,0 +1,171 @@
+---
+title: 'How to verify child chain state on parent chain'
+description: Learn how to verify child chain state on its parent chain
+user_story: As a developer, I want to understand how to verify child chain state on parent chain.
+content_type: how-to
+---
+
+Arbitrum implements fraud proof system. This system can ensure that the state of child chain will be safely maintained by parent chain. In this system, the validator will send the state information on child chain to parent chain from time to time.
+Used by the challenge system, those information is included in a structure called rblock and assertion (see the [documentation](../inside-arbitrum-nitro/inside-arbitrum-nitro.mdx#arbitrum-rollup-protocol) for details), so we can read and decode the rblock/assertion on parent chain to extract the information which can be used for the parent chain on-chain contract or off-chain tool validation.
+
+Before we begin, we will introduce the key component: rblock, assertion and send roots.
+
+# Rblock and Assertion
```

Comment:
There's some duplication between this content and [The Rollup Chain in Inside Arbitrum Nitro](https://docs.arbitrum.io/inside-arbitrum-nitro/#the-rollup-chain).

This is also explaining _how things work_, not _how you, dear developer, can do something_.

One option would be to:

1. Merge all of the information that explains "how it works" into the above-linked _Inside Arbitrum Nitro_ concept document (so we're not fragmenting this content across multiple docs that may drift apart as Nitro evolves).
2. Provide a condensed version of this upfront, linking to Inside Nitro.
3. Jump right into "How to verify a confirmed child chain block hash".

It could also be useful to explicitly state - when and why would a developer need to complete these steps? When do they need to care about manually verifying state? Is this a "just in case you don't trust the underlying protocol" thing? I imagine some developers may expect this to be baked into the protocol and effectively abstracted away from the happy path, so getting more precise about the "user story" above can help us align on a more precisely targeted content objective.

_As a developer who ______, I want to learn how to ________ when/because _____________._

I'll pause this review until we're aligned on the above nits 👍